ushahidi/platform-client
# Adding fixes from qa to location picker and confirm-before-delete-dialogs

This pull request makes the following changes:
- Closing the dropdown in the location-picker when clicking outside map
- Adding the text 'This action cannot be undone. Please proceed with caution.' to dialog before deleting.

Test these changes by:
- Start adding a post, search for a location, click outside the map or choose a location from list and the dropdown should close.
- Start deleting a post or survey, the dialog should contain the text 'This action cannot be undone. Please proceed with caution.'

Fixes ushahidi/platform#1528 #494  .

Ping @ushahidi/platform
